src/components/Dashboard.tsx

```diff
@@ -1,10 +1,48 @@
+import { useSelector } from 'react-redux'
+import { useFirestoreConnect } from 'react-redux-firebase'
+import { RootState } from '../store/store'
 import { Header } from './Header'
+import { ColumnList } from './ColumnList'
+import { ColumnForm } from './ColumnForm'
 
 export const Dashboard = () => {
+  const { uid } = useSelector((state: RootState) => state.firebase.auth)
+  useFirestoreConnect({
+    collection: `users/${uid}/columns`,
+    storeAs: 'columns',
+  })
+  const columns = useSelector(
+    (state: RootState) => state.firestore.data.columns
+  )
+  const columnsArray = Object.entries(columns ?? []).map(([id, column]) => ({
+    id,
+    ...column,
+  }))
+
+  useFirestoreConnect({
+    collection: `users/${uid}/cards`,
+    storeAs: 'cards',
+  })
+  const cards = useSelector((state: RootState) => state.firestore.data.cards)
```

Comment:
By the way, it would be great to keep selectors outside the components and just reuse them. Something like custom hooks. You can think about it, or we can work on that later 🙂